Block/BaseBlockService.php

```diff
@@ -16,82 +16,29 @@
 use Sonata\CoreBundle\Model\Metadata;
 use Sonata\CoreBundle\Validator\ErrorElement;
 use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
-use Symfony\Component\HttpFoundation\Response;
+
+@trigger_error(
+    'This class is deprecated since 3.x and will be removed with the 4.0 release.'.
+    'Use '.__NAMESPACE__.'\AbstractBlockService instead.', E_USER_DEPRECATED
```

Comment:
This is a multiline call, each argument must be on its own line , so please add a carriage return after the comma